frontend/src/container/GridGraphLayout/Graph/FullView/index.tsx

```diff
@@ -57,94 +52,34 @@ function FullView({
 		enum: widget?.timePreferance || 'GLOBAL_TIME',
 	});
 
-	const onFetchDataHandler = useCallback(async () => {
-		try {
-			const maxMinTime = GetMaxMinTime({
-				graphType: widget.panelTypes,
-				maxTime,
-				minTime,
-			});
-
-			const getMinMax = (
-				time: timePreferenceType,
-			): { min: string | number; max: string | number } => {
-				if (time === 'GLOBAL_TIME') {
-					const minMax = GetMinMax(globalSelectedTime);
-					return {
-						min: convertToNanoSecondsToSecond(minMax.minTime / 1000),
-						max: convertToNanoSecondsToSecond(minMax.maxTime / 1000),
-					};
-				}
-
-				const minMax = getStartAndEndTime({
-					type: selectedTime.enum,
-					maxTime: maxMinTime.maxTime,
-					minTime: maxMinTime.minTime,
-				});
-				return { min: parseInt(minMax.start, 10), max: parseInt(minMax.end, 10) };
-			};
-
-			const queryMinMax = getMinMax(selectedTime.enum);
-			const response = await Promise.all(
-				widget.query
-					.filter((e) => e.query.length !== 0)
-					.map(async (query) => {
-						const result = await getQueryResult({
-							end: queryMinMax.max.toString(),
-							query: query.query,
-							start: queryMinMax.min.toString(),
-							step: `${getStep({
-								start: queryMinMax.min,
-								end: queryMinMax.max,
-								inputFormat: 's',
-							})}`,
-						});
-						return {
-							query: query.query,
-							queryData: result,
-							legend: query.legend,
-						};
-					}),
-			);
-
-			const isError = response.find((e) => e.queryData.statusCode !== 200);
+	const onFetchDataHandler = useCallback(async (): Promise<void> => {
+		const getGlobalMinMax = (
+			time: timePreferenceType,
+		): GlobalMinMaxTime | undefined => {
+			if (time === 'GLOBAL_TIME') {
+				const minMax = GetMinMax(globalSelectedTime);
+				return {
+					min: convertToNanoSecondsToSecond(minMax.minTime / 1000),
+					max: convertToNanoSecondsToSecond(minMax.maxTime / 1000),
+				};
+			}
 
-			if (isError !== undefined) {
-				setState((state) => ({
-					...state,
-					error: true,
-					errorMessage: isError.queryData.error || 'Something went wrong',
-					loading: false,
-				}));
-			} else {
-				const chartDataSet = getChartData({
-					queryData: {
-						data: response.map((e) => ({
-							query: e.query,
-							legend: e.legend,
-							queryData: e.queryData.payload?.result || [],
-						})),
-						error: false,
-						errorMessage: '',
-						loading: false,
-					},
-				});
+			return undefined;
+		};
 
-				setState((state) => ({
-					...state,
-					loading: false,
-					payload: chartDataSet,
-				}));
-			}
-		} catch (error) {
-			setState((state) => ({
-				...state,
-				error: true,
-				errorMessage: (error as AxiosError).toString(),
-				loading: false,
-			}));
-		}
-	}, [widget, maxTime, minTime, selectedTime.enum, globalSelectedTime]);
+		// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
React identifies `useQuery` as a hook instead of a function (because of it's naming convention). Hence had to add the eslint-disable comment.
We can also change the function name to be `getQuery` to resolve this issue.